Dockerfile

```diff
@@ -218,7 +241,8 @@ RUN printf 'eval $(pd autocomplete:script bash)' >> ${HOME}/.bashrc.d/99-pdcli.b
     && bash -c "SHELL=/bin/bash pd autocomplete --refresh-cache"
```

Comment:
It looks like the version of node in the ubi-minimal image is, by default, lower than that required by PDCLI:

```
npm WARN notsup Unsupported engine for pagerduty-cli@0.0.82: wanted: {"node":">=16.0.0","npm":">=6.0.0"} (current: {"node":"10.24.0","npm":"6.14.11"})
npm WARN notsup Not compatible with your version of node/npm: pagerduty-cli@0.0.82
npm WARN notsup Unsupported engine for @oclif/command@1.8.16: wanted: {"node":">=12.0.0"} (current: {"node":"10.24.0","npm":"6.14.11"})
npm WARN notsup Not compatible with your version of node/npm: @oclif/command@1.8.16
```

Reply:
We could possibly use the nodejs 16 minimal image (https://catalog.redhat.com/software/containers/rhel8/nodejs-16-minimal/615aefc7c739c0a4123a87e2) as the base image, since it is itself based on ubi8-minimal, with just the latest node version added.

Reply:
That sounds fine, I'll look at the image later

Reply:
this migration is very annoying, as this image is a non-root image and I tried to comply with it
though the issue still remains with the newer version
```
Step 112/120 : RUN npm install -g pagerduty-cli@${PAGERDUTY_VERSION}
 ---> Running in fcae2c05f41f

added 220 packages, and audited 221 packages in 19s

20 packages are looking for funding
  run `npm fund` for details

4 moderate severity vulnerabilities

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
npm notice
npm notice New minor version of npm available! 8.1.2 -> 8.3.1
npm notice Changelog: <https://github.com/npm/cli/releases/tag/v8.3.1>
npm notice Run `npm install -g npm@8.3.1` to update!
npm notice
```

Reply:
WFM